charts/atlas-advanced/values.yaml

```diff
@@ -0,0 +1,41 @@
+secret:
+  orgID: ""
+  privateKey: ""
+  publicKey: ""
+
+project:
+  name: ""
+  cidrEntry: 0.0.0.0/0
+
+deployment:
+  regions:
+    # For AWS: "US_GOV_WEST_1" "US_GOV_EAST_1" "US_EAST_1" "US_EAST_2" "US_WEST_1" "US_WEST_2" "CA_CENTRAL_1" "EU_NORTH_1" "EU_WEST_1" "EU_WEST_2" "EU_WEST_3" "EU_CENTRAL_1" "EU_CENTRAL_2" "AP_EAST_1" "AP_NORTHEAST_1" "AP_NORTHEAST_2" "AP_NORTHEAST_3" "AP_SOUTHEAST_1" "AP_SOUTHEAST_2" "AP_SOUTHEAST_3" "AP_SOUTHEAST_4" "AP_SOUTH_1" "AP_SOUTH_2" "SA_EAST_1" "CN_NORTH_1" "CN_NORTHWEST_1" "ME_SOUTH_1" "ME_CENTRAL_1" "AF_SOUTH_1" "EU_SOUTH_1" "EU_SOUTH_2" "IL_CENTRAL_1" "CA_WEST_1" "GLOBAL"
```

Comment:
I see this is a comment. Do we plan to keep this up-to-date? If so, how? Can't we reference existing docs for valid values instead?

Reply:
Updated. Provided a link to the docs.